### PR TITLE
Documentation: "click at" replaced by "click on", see issue #681.

### PR DESCRIPTION
**Pull request by RainerKlute**

### CODE REVIEW

**Comment by RainerKlute** [comment]
@tkutz, please merge.

File: plugins/org.yakindu.sct.doc.user/src/Part2-User-Guide/40_editing_statecharts.textile [hunk 1]
```diff
@@ -61,8 +61,13 @@ The canvas is the statechart editor's drawing area. When you create a new statec
 
 The following list gives an overview of what kind of actions you can perform on the canvas:
 * Add or remove a region
+<<<<<<< HEAD
+** To add a region to the canvas, select _Region_ in the "editor palette":#editor-palette, then click on the canvas location you want to place the region.
+** To remove a region from the canvas, select the region, then press the @[Del]@ key or select _Edit → Delete_ from the menu bar.
+=======
 ** To add a region to the canvas, select _Region_ in the "editor palette":#editor-palette, then click at the canvas location you want to place the region.
 ** To remove a region from the canvas, select the region, then press @[Del]@ or select _Edit → Delete_ from the menu bar.
+>>>>>>> master
```
Comment (tkutz) [inline plugins/org.yakindu.sct.doc.user/src/Part2-User-Guide/40_editing_statecharts.textile]: This indicates a merge conflict on your side. Please resolve this conflict first!

File: plugins/org.yakindu.sct.doc.user/src/Part2-User-Guide/40_editing_statecharts.textile [hunk 2]
```diff
@@ -71,7 +76,11 @@ h3(#editor-palette). Editor palette
 
 The editor palette provides you with a set of various actions and statechart editing tools. By default, the palette is located right of the canvas, but you can also drag it to the left.
 
+<<<<<<< HEAD
+You can hide the palette by clicking at the small triangle in the palette's title bar. click on the triangle again to make the palette reappear.
+=======
 You can hide the palette by clicking the small triangle in the palette's title bar. Click the triangle again to make the palette reappear.
+>>>>>>> master
```
Comment (tkutz) [inline plugins/org.yakindu.sct.doc.user/src/Part2-User-Guide/40_editing_statecharts.textile]: This indicates a merge conflict on your side. Please resolve this conflict first!

File: plugins/org.yakindu.sct.doc.user/src/Part2-User-Guide/40_editing_statecharts.textile [hunk 3]
```diff
@@ -71,7 +71,7 @@ h3(#editor-palette). Editor palette
 
 The editor palette provides you with a set of various actions and statechart editing tools. By default, the palette is located right of the canvas, but you can also drag it to the left.
 
-You can hide the palette by clicking the small triangle in the palette's title bar. Click the triangle again to make the palette reappear.
+You can hide the palette by clicking at the small triangle in the palette's title bar. Click on the triangle again to make the palette reappear.
```
Comment (tkutz) [inline plugins/org.yakindu.sct.doc.user/src/Part2-User-Guide/40_editing_statecharts.textile]: Why "clicking **at** the small triangle" and next sentence "Click **on** the triangle". Please be consistent. Either use "on" or leave the preposition out.

Comment (RainerKlute) [inline plugins/org.yakindu.sct.doc.user/src/Part2-User-Guide/40_editing_statecharts.textile]: Oops!

**Pull request merged**